eosiojava/build.gradle

```diff
@@ -36,12 +36,12 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:2.26.0'
     api 'org.bitcoinj:bitcoinj-core:0.15.2'
-    api 'org.slf4j:slf4j-jdk14:1.7.25'
+    implementation 'io.reactivex.rxjava2:rxjava:2.2.19'
 }
 
 def libraryGroupId = 'one.block'
 def libraryArtifactId = 'eosiojava'
-def libraryVersion = '0.1.2'
+def libraryVersion = '0.2.0-amqp9'
 
```

Comment:
what version do we actually want to bump this to?

Reply:
Undetermined at this point.  Currently the next release is targeted toward eosio-2.1 features and amqp support is not scheduled to be part of that release.  If there is a need for it @brandonfancher should be consulted to put it into the release plan.

Reply:
It is needed to support our AMQP provider/plug-in, and we plan to use that on services in the near future so would like to merge this in @brandonfancher @opi-smccoole